cpp/scripts/run-cmake-format.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+# This script is a wrapper for cmakelang that may be used with pre-commit. The
```

Comment:
separate PR?

Reply:
Oops, accidentally commited this file in this PR. Reveted.